support/Pipelines/Bbh/ControlId.py

```diff
@@ -0,0 +1,173 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+from pathlib import Path
+from typing import Optional, Union
+
+import h5py
+import numpy as np
+import yaml
+
+from spectre.Pipelines.Bbh.InitialData import generate_id
+
+DEFAULT_RESIDUAL_TOLERANCE = 1.0e-6
+DEFAULT_MAX_ITERATIONS = 30
+
+
+def control_id(
+    id_input_file_path: Union[str, Path],
+    id_run_dir: Optional[Union[str, Path]] = None,
+    residual_tolerance: float = DEFAULT_RESIDUAL_TOLERANCE,
+    max_iterations: int = DEFAULT_MAX_ITERATIONS,
+):
+    """Control BBH physical parameters.
+
+    This funciton is called after initial data has been generated and horizons
+    have been found in 'PostprocessId.py'. It uses an iterative scheme to drive
+    the black hole physical parameters (masses and spins) closer to the desired
+    values.
+
+    For each iteration, this function does the following:
+
+    - Determine new guesses for the Superposed Kerr-Schild (SKS).
```

Comment:
I think this is not specific to SKS. It should also work for other ID gauges if we adjust the relationship between masses/spins and horizon radii/frequencies in `id_parameters`. So reword to something like: "Determine new guesses for ID input parameters"